web-app/js/portal/ui/Viewport.js

```diff
@@ -7,6 +7,21 @@
 
 Ext.namespace('Portal.ui');
 
+window.onbeforeunload = function() {
+
+    if (   Portal.config.isFeatureToggleEnabled('chatOnFailedSearch')
+        && (Portal.ui.MainPanel.maxStepReached < TAB_INDEX_VISUALISE)) {
+        // Replace this with a better UI...
+        return String.format(
+            "It looks like you didn't find what you were looking for. Want to chat - go to this link: {0}",
```

Comment:
I like the first part of the sentence. but maybe for the second we can have:
"Would you like to chat with us? Follow this link."